### ISSUE TITLE
High Scores from TXT files are off by a line

### ISSUE THREAD

**Issue by rubadub-github** [syd711/vpin-studio#20]
VPX 10.7.4 x64.

off by a line - 
8 Ball (Williams 1966) 1.0.0f
2001 (Gottlieb 1971) 0.99

correct - 
300 (Gottlieb 1975) 1.0.0f

I cleared the TXT and started the tables so they reset and this is what you get.

both tables started with this.
```
HSScore(1) = 500
HSScore(2) = 400
HSScore(3) = 350
HSScore(4) = 300
HSScore(5) = 250

HSName(1) = "AAA"
HSName(2) = "ZZZ"
HSName(3) = "XXX"
HSName(4) = "ABC"
HSName(5) = "BBB"
```

Looks like somewhere along the way tables added or removed a line in the top, so some scores start on line 6, others line 7.  

Looks like there are differences in the sub savehs.

![8ball](https://github.com/syd711/vpin-studio/assets/68559237/57177d6e-339d-462d-a5a3-5759358069fa)
![300](https://github.com/syd711/vpin-studio/assets/68559237/6f3c83ae-c07e-4014-a674-9f03063403be)

**Comment by syd711** [comment]
Thank you for the hint. I will look into it today.

**Comment by syd711** [comment]
Fixed, bugfix will come 1.4.3.
